### ISSUE TITLE
Divinity 2 Animations not imported as expected

### ISSUE THREAD

**Issue by X4NDR0** [niftools/blender_niftools_addon#575]
Hi, Candoran fixed last problem with the same game and same animations by changing scale to 1.0 while importing.

Here is the picture of the problem : https://prnt.sc/u5L6KnqbpS56
Error occurs in blender plugin and nifskope.

Animation name : CM_Ground_Sleep_03 (There is more with same or similiar problem)
NIF File : https://www.sendspace.com/file/qqlct4

**Comment by Candoran2** [comment]
No, not yet. Also, it seems like the files are no longer available. Could you attach them as a zip file (drag and drop into the issue) instead of using that service? That way it remains available.
